RELEASES.md

```diff
@@ -5,7 +5,8 @@
 ... 
 
 ## [v0.7.6](https://github.com/ava-labs/subnet-evm/releases/tag/v0.7.5)
-@TODO - write release notes here on what is in this release
+- Demoted unnecessary error log in `core/txpool/legacypool.go` to warning, displaying unexpected but valid behavior
+- @TODO - write release notes here on what is in this release
```

Comment:
we should clean this up before releasing (should be in the same PR that bumps avalanhchego to v1.13.3 once it's out)